utils/dchemigen/dchemigen.c

```diff
@@ -0,0 +1,185 @@
+#include <stdio.h>
```

Comment:
Needs KOS copyright header.

Reply:
This one I can say I worked on and add a header to, definitely.